proto/frostdb/wal/v1alpha1/wal.proto

```diff
@@ -17,12 +17,27 @@ message Record {
 message Entry {
   // The write-type entry.
   message Write {
+    reserved 3; // Previously used for Arrow indication, but all writes are now Arrow.
     // Table name of the write.
     string table_name = 1;
     // Data is the data of the write.
     bytes data = 2;
-    // Arrow indicates that data is stored in arrow record format.
-    bool arrow = 3;
+    // Tx is the transaction ID of the write.
+    uint64 tx = 4;
```

Comment:
I don't think it's necessary to add the transaction ID into the Write message. The transaction ID is already implicitly the WAL index.